api/lib/mno_enterprise/concerns/mailers/system_notification_mailer.rb

```diff
@@ -32,12 +32,21 @@ def default_sender
   #   :confirmation_link
   #
   def confirmation_instructions(record, token, opts={})
-    template = record.confirmed? && record.unconfirmed_email? ? 'reconfirmation-instructions' : 'confirmation-instructions'
+    update_email = record.confirmed? && record.unconfirmed_email?
```

Comment:
Could you add more comments here to explain what's going on?
Either on the method doc or inline